cpp/include/gpu_operations.h

```diff
@@ -47,6 +56,54 @@ class GpuOperations {
   static Matrix<T> OuterProduct(const Vector<T> &a, const Vector<T> &b);
 };
 
+template <typename T>
+Matrix<T> GpuOperations<T>::Multiply(const Matrix<T> &a, const Matrix<T> &b) {
+  if (a.cols() == b.rows()) {
+  int m = a.rows();
```

Comment:
The wildcard would only make sure that the Makefile contains the *.cc files at the time cmake .. is called. So if you add a new file after running cmake .., the Makefile generated before that new file is added will not include the added file. When you start from scratch, are those affected files in develop already?